mcs/class/System.Net.Http/NSUrlSessionHandler.cs

```diff
@@ -592,7 +592,8 @@ class NSUrlSessionDataTaskStreamContent : StreamContent
 		{
 			Action disposed;
 
-			public NSUrlSessionDataTaskStreamContent (NSUrlSessionDataTaskStream source, Action onDisposed, CancellationToken token) : base (source, token)
+			[Obsolete ("FIXME: Please talk to Martin about this; see https://github.com/mono/mono/issues/12996.")]
```

Comment:
This is private API used in this file only in our source code, if you need to fix it then please do it

Reply:
That's why this is still a _Draft_ pull request ;-)